src/main/java/com/unb/tracker/repository/CourseRepository.java

```diff
@@ -2,12 +2,17 @@
 
 
 import com.unb.tracker.model.Course;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 
 import java.util.List;
 
 public interface CourseRepository extends CrudRepository<Course, Long> {
     public List<Course> findByInstructorUsernameAndNameAndSection(String username, String name, String section);
 
     public List<Course> findByInstructorUsernameAndName(String username, String name);
+
+    @Query(value = "SELECT * FROM Course WHERE course.name LIKE CONCAT('%',:pname,'%') OR course.section LIKE CONCAT('%',:pname,'%') LIMIT 10", nativeQuery = true)
```

Comment:
I'm getting an SQL Error here because the table on my system is named `course` vs the `Course` that is written here. Maybe a `@table(Name="...")` annotation would be appropriate

Reply:
That's super weird. I'm not having that issue at all. I'll give that a go.

Reply:
Should be fixed now.